File: meetings/meeting-management/create-meetings/code-samples/createMeeting.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# PATH PARAMETERS
extensionId = '<ENTER VALUE>'
accountId = '<ENTER VALUE>'

# POST BODY
body = {
    'topic': '<ENTER VALUE>',
    'meetingType': 'Scheduled',
    'schedule': {
        'startTime': '<ENTER VALUE>',
        'durationInMinutes': 000,
        'timeZone': {
            'uri': '<ENTER VALUE>',
            'id': '<ENTER VALUE>',
            'name': '<ENTER VALUE>',
            'description': '<ENTER VALUE>'
        }
    },
    'password': '<ENTER VALUE>',
    'host': {
        'id': '<ENTER VALUE>'
    },
    'allowJoinBeforeHost': true,
    'startHostVideo': true,
    'startParticipantsVideo': true,
    'usePersonalMeetingId': true,
    'audioOptions': [
        '<ENTER VALUE>'
    ]
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.post(f'/restapi/v1.0/account/{accountId}/extension/{extensionId}/meeting', body)
# PROCESS RESPONSE

File: meetings/meeting-management/create-meetings/code-samples/createMeeting.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$extensionId = '<ENTER VALUE>';
$accountId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'topic' => '<ENTER VALUE>',
    'meetingType' => 'Scheduled',
    'schedule' => array(
        'startTime' => '<ENTER VALUE>',
        'durationInMinutes' => 000,
        'timeZone' => array(
            'uri' => '<ENTER VALUE>',
            'id' => '<ENTER VALUE>',
            'name' => '<ENTER VALUE>',
            'description' => '<ENTER VALUE>'
        )
    ),
    'password' => '<ENTER VALUE>',
    'host' => array(
        'id' => '<ENTER VALUE>'
    ),
    'allowJoinBeforeHost' => true,
    'startHostVideo' => true,
    'startParticipantsVideo' => true,
    'usePersonalMeetingId' => true,
    'audioOptions' => array(
        '<ENTER VALUE>'  
    )
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/account/{$accountId}/extension/{$extensionId}/meeting", $body);
// PROCESS RESPONSE
?>
